src/storiesDefault/Button.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './button.css';
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({primary, backgroundColor, size, label, ...props}) => {
+  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
```

Comment:
To delete `storiesDefault` / example package after our storybooks have the same features.